scripts/tools/initialize_virtualenv.py

```diff
@@ -111,7 +111,7 @@ def main():
     """
     Setup local virtual environment.
     """
-    airflow_home_dir = os.environ.get("AIRFLOW_HOME", Path.home() / "airflow")
+    airflow_home_dir = Path(os.environ.get("AIRFLOW_HOME", Path.home() / "airflow"))
     airflow_sources = str(Path(__file__).parents[2])
```

Comment:
I think it's safest for this to be a Path object as well, instead of casting to a string. A few lines below `airflow_home_dir` and `airflow_sources` are compared for equality, which I think is bound to always return False when comparing a string between a Path. We should have a unit test covering this as well.

Reply:
Yep noticed the same before the comment of yours :)